src/sql/azdata.proposed.d.ts

```diff
@@ -261,7 +261,8 @@ declare module 'azdata' {
 		iconWidth?: string;
 		iconHeight?: string;
 		selectedCardId?: string;
-		orientation?: Orientation // Defaults to horizontal
+		orientation?: Orientation; // Defaults to horizontal
+		cardStyle?: 'regular' | 'portalCard'; // Defaults to regular
```

Comment:
I am not a fan of the name portalcard. the difference between the old and new is the icon position, maybe we can call this property: IconPosition: top|left?

Reply:
Yeah, sounds good to me. I couldn't think of a better while I was coding.

Reply:
cool, and don't forgot to do a bulk find/replace of the css class names

Reply:
I agree with not calling it a portal card :)